resources/js/features/games/components/TopAchieversMainRoot/TopAchieversMainRoot.test.tsx

```diff
@@ -0,0 +1,155 @@
+import { router } from '@inertiajs/react';
+import userEvent from '@testing-library/user-event';
+
+import { render, screen } from '@/test';
+import { createFactory } from '@/test/createFactory';
+import { createGame, createPaginatedData, createSystem, createUser } from '@/test/factories';
+
+import { TopAchieversMainRoot } from './TopAchieversMainRoot';
+
+export const createAchiever = createFactory<App.Platform.Data.GameTopAchiever>((faker) => {
+  return {
+    rank: 1,
+    user: createUser(),
+    score: faker.number.int({ min: 1, max: 100 }),
+    badge: null,
+  };
+});
```

Comment:
See other feedback related to creating a common factory in `@/test/factories`.

Reply:
done.